x-pack/plugins/ml/public/application/timeseriesexplorer/timeseriesexplorer.js

```diff
@@ -392,13 +426,11 @@ export class TimeSeriesExplorer extends React.Component {
       zoom,
     } = this.props;
 
-    const { loadCounter: currentLoadCounter } = this.state;
+    const { loadCounter: currentLoadCounter, functionDescription } = this.state;
 
-    const currentSelectedJob = mlJobService.getJob(selectedJobId);
+    const functionToPlotByIfMetric = aggregationTypeTransform.toES(functionDescription);
 
-    if (currentSelectedJob === undefined) {
-      return;
-    }
```

Comment:
Can you explain why this check is no longer necessary?

Reply:
Thanks for catching that. This was removed by accident. I have revert the change in https://github.com/elastic/kibana/pull/81662/commits/64ecedbd1dccaa5b916c4e81538729a17dfb3afb